docs/src/remap/remap.adoc

```diff
@@ -2343,15 +2343,15 @@ FIXME Write missing information
 
 FIXME Write missing information
 
-=== A diagram: task, interp, iocontrol, UI (??)
+=== A diagram: `task`, `interp`, `iocontrol`, `UI` (??)
```

Comment:
Not sure if we should have monospaced font in headings. Maybe something for the style guide ;-)

Reply:
#1530